azkaban-execserver/src/main/resources/execute-as-user.c

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2015 LinkedIn Corp.
```

Comment:
Given that this is built as part of Azkaban and given that Gradle supports building native code, I think building this C file should be handled by the build system and the user should not need to worry about building this file separately.